### PR TITLE
After removing overlays from Swift repo we are started seeing followi…

### PR DESCRIPTION
**Pull request by shahmishal**
…ng failures

* lldb-api :: lang/swift/foundation_value_types/indexpath/TestSwiftFoundationTypeIndexPath.py
* lldb-shell :: SwiftREPL/FoundationTypes.test
* lldb-shell :: SwiftREPL/ResilientArray.test
* lldb-shell :: SwiftREPL/ResilientDict.test

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

File: lldb/test/API/lang/swift/foundation_value_types/indexpath/TestSwiftFoundationTypeIndexPath.py [hunk 1]
```diff
@@ -13,4 +13,4 @@
 from lldbsuite.test.decorators import *
 
 lldbinline.MakeInlineTest(__file__, globals(),
-                          decorators=[swiftTest,skipIf(oslist=['windows', 'linux'])])
+                          decorators=[swiftTest,skipIf(oslist=['macosx', 'windows', 'linux'])])
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/foundation_value_types/indexpath/TestSwiftFoundationTypeIndexPath.py]: `decorators=[swiftTest,skipIf(oslist=['windows', 'linux'], expectedFailureAll(oslist=["macosx"], bugnumber='rdar://79052960')]`

File: lldb/test/Shell/SwiftREPL/FoundationTypes.test [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 // REQUIRES: system-darwin
 // REQUIRES: swift
+// REQUIRES: rdar79052960
```
Comment (adrian-prantl) [inline lldb/test/Shell/SwiftREPL/FoundationTypes.test]:
```
// XFAIL: system-darwin
// rdar://79052960
```

File: lldb/test/Shell/SwiftREPL/ResilientArray.test [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // REQUIRES: system-darwin
 // REQUIRES: swift
+// REQUIRES: rdar79053073
```
Comment (adrian-prantl) [inline lldb/test/Shell/SwiftREPL/ResilientArray.test]:
```
// XFAIL: system-darwin
// rdar://79052960
```

File: lldb/test/Shell/SwiftREPL/ResilientDict.test [hunk 4]
```diff
@@ -1,5 +1,6 @@
 // REQUIRES: system-darwin
 // REQUIRES: swift
+// REQUIRES: rdar79053148
```
Comment (adrian-prantl) [inline lldb/test/Shell/SwiftREPL/ResilientDict.test]:
```
// XFAIL: system-darwin
// rdar://79052960
```

**Comment by adrian-prantl** [comment]
Preferably we XFAIL the tests, so we get notified when their status changes.

**Comment by adrian-prantl** [comment]
We can also merge this and change it to XFAILs in a follow-up commit

**Comment by shahmishal** [comment]
@swift-ci test

**Comment by adrian-prantl** [comment]
```
03:05:55 Unexpectedly Passed Tests (4):
03:05:55   lldb-api :: lang/swift/foundation_value_types/indexpath/TestSwiftFoundationTypeIndexPath.py
03:05:55   lldb-shell :: SwiftREPL/FoundationTypes.test
03:05:55   lldb-shell :: SwiftREPL/ResilientArray.test
03:05:55   lldb-shell :: SwiftREPL/ResilientDict.test
```

Is that because it wasn't tested against the PR that removes the overlays?

**Comment by shahmishal** [comment]
Yes, its because it was not tested with swift PR. I will merge both PRs together.

**Pull request merged**